proposals/lip-0045.md

```diff
@@ -24,7 +24,8 @@ Interoperability has the potential to solve the scaling issue of blockchains: In
 
 To achieve interoperability, chains in the Lisk ecosystem (the mainchain and the sidechains participating in interoperability) exchange information via _cross-chain transactions_. As an example, a user can issue a cross-chain transaction to transfer part of its mainchain LSK balance to a sidechain, and subsequently use it to issue one of the sidechain custom commands.
 
-In the Lisk protocol, cross-chain transactions are special in that, after being included in their origin ledger, some new data structures, the _cross-chain messages_ (CCMs), are created, which are transmitted to other chains via special commands called _cross-chain update_ (CCU) commands. CCUs also contain information about the chain, in the form of a _certificate_.
+In the Lisk protocol, cross-chain transactions are special in that, after being included in their origin ledger, some new data structures, the _cross-chain messages_ (CCMs), are created, which are transmitted to other chains via special commands called _cross-chain update_ (CCU) commands. CCUs also contain information about the chain, in the form of a _certificate_. 
+In the receiving chain, CCMs trigger the execution of _cross-chain commands_, in a way similar to how transaction trigger the execution of regular commands.
 
 One of the main motivations for proposing a [certificate-based communication](https://lisk.com/blog/research/introduction-blockchain-interoperability) between chains is the ability for all chains in the ecosystem to remain state machines. This means that all information needed to validate and advance a chain to the next block is present on the chain itself.  In the Lisk interoperability solution, the only information that needs social consensus is information about the initial validators of the other chain. On the mainchain, this information is received via the sidechain registration command; on sidechains, in a similar way, via the mainchain registration command.
 
```

Comment:
I cannot directly comment on the line below, but you write

> The name and network ID of the mainchain are global protocol constants in the whole ecosystem.

Here the chain ID of the Lisk Mainchain is also a global protocol constant, maybe even more important than the name. Chain names in principle would not need to be stored on the sidechains.

Reply:
mh, can you suggest an alternative? I find it clear to be honest

Reply:
I think the current formulation is also fine, I resolve this comment.